tests/network-setups/indy/docker/run_indy_nodes.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/src/indy_projects/ursa/target/release/
+generate_indy_pool_transactions --nodes 4 --clients 5 --nodeNum 1 2 3 4
+
+start_indy_node Node1 0.0.0.0 9701 0.0.0.0 9702 &
+
+start_indy_node Node2 0.0.0.0 9703 0.0.0.0 9704 &
+
+start_indy_node Node3 0.0.0.0 9705 0.0.0.0 9706 &
+
+start_indy_node Node4 0.0.0.0 9707 0.0.0.0 9708 &
```

Comment:
Is it possible to run the above in a loop so we can start up an arbitrary number of nodes? Is there a limitation in Indy that we should be aware of that requires us to run exactly 4 nodes?